cap/modules/records/serializers/schemas/common.py

```diff
@@ -83,6 +87,42 @@ def check_unknown_fields(self, data, original_data):
                     )
 
 
+class CAPObjectVersionSchema(ObjectVersionSchema):
+    """Schema for ObjectVersions."""
+
+    bucket = fields.UUID(attribute='bucket_id', dump_only=True)
+
+    def dump_links(self, o):
```

Comment:
```suggestion
    def dump_links(self, obj/object/file_object/file_obj):
```

Better var name would be helpful for later.